src/cli.c

```diff
@@ -208,11 +208,13 @@ const clivalue_t valueTable[] = {
     { "failsafe_throttle", VAR_UINT16, &cfg.failsafe_throttle, 1000, 2000 },
     { "failsafe_detect_threshold", VAR_UINT16, &cfg.failsafe_detect_threshold, 100, 2000 },
     { "auto_disarm_board", VAR_UINT8, &mcfg.auto_disarm_board, 0, 60 },
-    { "rssi_aux_channel", VAR_INT8, &mcfg.rssi_aux_channel, 0, 4 },
+    { "rssi_aux_channel", VAR_INT8, &mcfg.rssi_aux_channel, 0, 14 },
     { "rssi_aux_max", VAR_UINT16, &mcfg.rssi_aux_max, 0, 1000 },
     { "rssi_adc_channel", VAR_INT8, &mcfg.rssi_adc_channel, 0, 9 },
     { "rssi_adc_max", VAR_INT16, &mcfg.rssi_adc_max, 1, 4095 },
     { "rssi_adc_offset", VAR_INT16, &mcfg.rssi_adc_offset, 0, 4095 },
+    { "rc_channel_count", VAR_UINT8, &mcfg.rc_channel_count, 8, 18 },
+    { "rc_aux_count", VAR_UINT8, &mcfg.rc_aux_count, 4, 8 },

```

Comment:
ok, I ended up getting rid of the fine-tunable aux count variable & combined it with the rc_channel_count functionality. The aux channel count will be calculated on boot, (rc_channel_count - 4), and constrained between 4 and 8.